MAUI/ImageEditor/save.md

```diff
@@ -215,4 +215,30 @@ The [`ImageSaved`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.ImageEdit
 
 {% endtabs %}
 
+### Save picker opening event
+
+The `SavePickerOpening` event occurs while the save picker is opening on the save icon click.
+Cancel: We can restrict the save opening by setting the Cancel argument to true. If save picker is disabled, the image will be saved in the default location.
```

Comment:
The `SavePickerOpening` event occurs while the save picker is opening on the save icon click.
`Cancel`: We can restrict the save picker opening by setting the `Cancel` argument to `true`. If the save picker is disabled, the image will be saved in the default location.

Reply:
Updated.